README.md

```diff
@@ -96,7 +96,17 @@ A `preHandler` to be applied on all routes. Useful for performing actions before
 
 ### proxyPayloads
 
-When you set this option as false, you will be able to access the body but it would also disable the direct pass through of the payload. The implementor will have to take care if the payload is proxied correctly.
+When you set this option as `false`, you will be able to access the body but it would also disable the direct pass through of the payload. The implementor will have to take care if the payload is parsed and proxied correctly.
+
+For Example,
+
+If you are expecting payload of type application/xml, then you would have to add a parser for it like this.
+
```

Comment:
```suggestion
When this option is `false`, you will be able to access the body but it will also disable direct pass through of the payload. As a result, it is left up to the implementation to properly parse and proxy the payload correctly.

For example, if you are expecting a payload of type `application/xml`, then you would have to add a parser for it like so:

```